### PR TITLE
2016 02 newsletter

### PR DESCRIPTION
**Pull request by alaindomissy**
first draft

### CODE REVIEW

File: content/posts/2016-02-01_february-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+Title: February Newsletter
+Date: 2016-01-04
```
Comment (macro1) [inline content/posts/2016-02-01_february-newsletter.md]: This needs to be updated. :grinning:

**Comment by davidfischer** [comment]
Do you want to mention in the opening paragraph that we are planning a Django Girls workshop for April and we are looking for volunteers?

File: content/posts/2016-02-01_february-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+at the Ansir Innovation Center (12:30pm - 3pm) The goal of the Saturday study group is to
+learn Python together in a collaborative environment. The study group is open
+to people of all skill levels and is particularly open to those new to Python.
+Bring a laptop!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/227667838/
+
+
+[PyLadies Casual Study Group][] (meetup.com) <br />
+San Diego PyLadies are meeting at Eclipse Chocolat (now a usual location!!)
```
Comment (macro1) [inline content/posts/2016-02-01_february-newsletter.md]: We can probably drop the 'now a usual location!!' bit. They've been there for the past 7 or 8 of their Tuesday meetings.

File: content/posts/2016-02-01_february-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+on Tuesday February 9 at 6pm. Get some of the best chocolate in San Diego and
```
Comment (macro1) [inline content/posts/2016-02-01_february-newsletter.md]: February 9th and 23rd, they meet twice a month.

**Comment by davidfischer** [comment]
If you want a possible news item, feel free to post about former-San Diegan Bunnie Hwang's new book about sourcing hardware in Shenzhen: https://www.crowdsupply.com/sutajio-kosagi/the-essential-guide-to-electronics-in-shenzhen

It's an instant classic for hardware hackers looking to get into a larger production cycle.

**Comment by davidfischer** [comment]
I don't see any movement from the Python BC guys so I guess there's nothing to announce. https://pythonbc.com/

**Comment by treyhunner** [comment]
^ @JesusAnaya @PythonBC do you have a meetup for the newsletter? Want to write a couple sentences? :smile_cat: :question:

**Comment by macro1** [comment]
Closing, because it's March.

@davidfischer the Shenzhen hardware thing looks really cool. Think I should include it in the March newsletter?

**Pull request closed without merging**